packages/core/src/server.ts

```diff
@@ -599,7 +599,7 @@ export class Controller extends API {
         // Add device to trusted devices
         if (
             this.context.device &&
-            !auth.trustedDevices.some(({ id }) => equalCT(id, this.context.device!.id)) &&
+            !auth.trustedDevices.some(({ id }) => id === this.context.device!.id) &&
```

Comment:
We don't actually need a constant-time comparison here, since we're already authenticated and both values are (in theory) already known to the caller.